plugins/modules/user_role_workflow_manager.py

```diff
@@ -1056,28 +1056,27 @@ def valid_user_config_parameters(self, user_config):
         """
         self.log("Validating user configuration parameters...", "INFO")
         error_messages = []
+        regex_name_validation = re.compile(r"^[A-Za-z0-9_-]+$")
+        regex_name_validation_msg = "must only contain letters, numbers, underscores and hyphens and should not contain spaces or other special characters."
 
-        first_name_regex = re.compile(r"^[A-Za-z0-9_-]+$")
-        self.validate_string_field(user_config.get("first_name"), first_name_regex, "first_name: 'first_name' must only contain letters, numbers, \
-                                   underscores, and hyphens and should not contain spaces or other special characters.", error_messages)
+        self.validate_string_field(user_config.get("first_name"), regex_name_validation,
+                                   "first_name: '{0}' {1}".format(user_config.get("first_name"), regex_name_validation_msg), error_messages)
 
-        last_name_regex = re.compile(r"^[A-Za-z0-9_-]+$")
-        self.validate_string_field(user_config.get("last_name"), last_name_regex, "last_name: 'last_name' must only contain letters, numbers, \
-                                   underscores, and hyphens and should not contain spaces or other special characters.", error_messages)
+        self.validate_string_field(user_config.get("last_name"), regex_name_validation,
+                                   "last_name: '{0}' {1}".format(user_config.get("last_name"), regex_name_validation_msg), error_messages)
 
         email_regex = re.compile(r"[^@]+@[^@]+\.[^@]+")
+        email_regex_msg = "email: Invalid email format for 'email': {0}".format(user_config.get("email"))
```

Comment:
email_regex_msg = "email: Invalid email format for 'email': {0}".format(email)

Reply:
Updated the code.